app/models/user.rb

```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
+end
```

Comment:
Kindly associate the model as well by adding the key_word has_many and belongs_to...